test/RelayHubPenalizations.test.ts

```diff
@@ -260,101 +284,59 @@ contract('RelayHub Penalizations', function ([_, relayOwner, relay, otherRelay,
       })
     })
 
-    describe('penalizable relay states', async function () {
+    describe('penalizable relay states', function () {
       context('with penalizable transaction', function () {
-        let penalizableTxData
-        let penalizableTxSignature
+        let penalizableTxData: string
+        let penalizableTxSignature: string
 
         beforeEach(async function () {
           // Relays are not allowed to transfer Ether
-          const { transactionHash } = await send.ether(relay, other, ether('0.5'));
+          const { transactionHash } = await send.ether(thirdRelayWorker, other, ether('0.5'));
           ({
             data: penalizableTxData,
             signature: penalizableTxSignature
           } = await getDataAndSignatureFromHash(transactionHash, chainId))
         })
 
         // All of these tests use the same penalization function (we one we set up in the beforeEach block)
-        function penalize () {
-          return expectPenalization((opts) => relayHub.penalizeIllegalTransaction(penalizableTxData, penalizableTxSignature, opts))
+        async function penalize (): Promise<TransactionResponse> {
+          return expectPenalization(async (opts) => relayHub.penalizeIllegalTransaction(penalizableTxData, penalizableTxSignature, opts))
         }
 
-        // Checks that a relay can be penalized, but only once
-        function testUniqueRelayPenalization () {
-          it('relay can be penalized', async function () {
-            await penalize()
+        context('with not owned relay worker', function () {
+          it('account cannot be penalized', async function () {
+            await expectRevert(penalize(), 'Unknown relay worker')
           })
+        })
 
-          it('relay cannot be penalized twice', async function () {
```

Comment:
seems like many test were removed here.

Reply:
These are same tests nested in different 'contexts'. Like, not staked -> not registered -> all good -> unregistered, etc. Some of these states no longer exist.